packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.vue

```diff
@@ -0,0 +1,90 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { useRipple, type RippleConfig } from "../../composables/useRipple";
+import { type OnyxRippleProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxRippleProps>(), {
+  enterDuration: "300ms",
+  leaveDuration: "100ms",
+  color: "var(--onyx-ripple-color, var(--onyx-color-base-primary-600))",
+  terminateOnPointerUp: false,
+});
+
+const config = computed<RippleConfig>(() => ({
+  color: props.color,
+  container: rippleTrigger,
+  terminateOnPointerUp: props.terminateOnPointerUp,
+}));
+
+const rippleTrigger = ref<HTMLElement>();
+
+const { ripples, hideRipple, events } = useRipple(config);
+</script>
+
+<template>
+  <span ref="rippleTrigger" class="onyx-ripple" v-on="events">
+    <transition-group name="onyx-ripple" @after-enter="hideRipple($event as HTMLElement)">
+      <span
+        v-for="[key, r] in ripples"
```

Comment:
Why are there multiple riples? One should be sufficient :)

Reply:
it's inspired by Material designs ripples, it feels good that you get a fresh ripple on every interaction 🙂. Most of the time there won't be more than 1 visible at a time.